holidays/be.yaml

```diff
@@ -1,5 +1,4 @@
 ---
-
 # https://nl.wikipedia.org/wiki/Feestdagen_in_Belgi%C3%AB
```

Comment:
You don’t like the empty line to separate the YAML header? When I run the Unix command `head -n 2 *.yaml` over the directory, it shows me that all files except `ch.yaml` have an empty line. If you want to change it, you can change it for all files for consistency. I don’t have a strong preference.

Reply:
That was a little accident. I readde the newline and added the newline in ch.yaml as well